web/server/api/endpoints/plan.py

```diff
@@ -24,7 +25,7 @@ async def run_plan(
     environment: t.Optional[str] = Body(None),
     plan_dates: t.Optional[models.PlanDates] = None,
     plan_options: models.PlanOptions = models.PlanOptions(),
```

Comment:
It's outside the scope of your changes, but I just noticed it. Can you please add a line like `plan_options = plan_options or models.PlanOptions()` below?
```suggestion
    plan_options: models.PlanOptions = None,
```